File: Uebung7BINT20/app/src/main/java/com/example/uebung7/MainActivity.java
package com.example.uebung7;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClickButton1(View v) {
        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("telekom://activity2"));
        startActivity(intent);
    }
}

File: uebung6_3BINT20/app/src/main/java/com/dasgrau/uebung6_3/MainActivity.kt
package com.dasgrau.uebung6_3

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Environment
import android.view.View
import java.io.File
import java.io.FileReader

/**
 * Basiert auf https://github.com/mercie-nyakio/Android-preferences
 */
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun onClickButtonSettings(v: View) {
        startActivity(Intent(this, SettingsActivity::class.java))
    }
}